addons/mail/static/src/models/blur_manager.js

```diff
@@ -24,7 +24,7 @@ registerModel({
     lifecycleHooks: {
         _willDelete() {
             this.video.removeEventListener('loadeddata', this._onVideoPlay);
-            this.selfieSegmentation.reset();
+            this.selfieSegmentation?.reset();
```

Comment:
I'm not sure but I don't think we could use the operator in 16.

Reply:
If accepted by linter at that version, then it's ok to use. At time of release of 16.0 we didn't do optional chaining at time of release because not necessarily supported by all browsers, but now that it is the case there's nothing preventing it. Bug fixes should be minimal and to the point, optional chaining helps quite a lot in that regard